tests/acceptance/features/apiSpaces/quota.feature

```diff
@@ -122,3 +122,20 @@ Feature: State of the quota
       | /filesForUpload/lorem-big.txt | /ocs/v1.php/cloud/users/%username% | 100      | 91.17          |
       | /filesForUpload/lorem.txt     | /ocs/v2.php/cloud/users/%username% | 200      | 6.99           |
       | /filesForUpload/lorem-big.txt | /ocs/v2.php/cloud/users/%username% | 200      | 91.17          |
+
+
+  @env-config
+  Scenario: upload a file by setting OCIS spaces max quota
+    Given "OCIS_SPACES_MAX_QUOTA" has been enabled and set to "10"
```

Comment:
I kinda like this step from https://github.com/owncloud/ocis/pull/6409/files
`Given the config "OCIS_SHARING_PUBLIC_WRITEABLE_SHARE_MUST_HAVE_PASSWORD" has been set to "true"`